tools/run_tests/xds_k8s_test_driver/tests/url_map/csds_test.py

```diff
@@ -37,6 +39,12 @@
 
 class TestBasicCsds(xds_url_map_testcase.XdsUrlMapTestCase):
 
+    @staticmethod
+    def is_supported(config: skips.TestConfig) -> bool:
+        if config.client_lang == _Lang.NODE:
+            return not config.version_lt('v1.5.x')
```

Comment:
How about:

```py
return config.version_gte('v1.5.x')
```

Reply:
I wrote it this way to be consistent with the checks @sergiitk added in #29925.

Reply:
If we can simplify the condition, I would recommend to do it. But it's not a hard blocker in any way.

Reply:
Yea, I'm fine one way or another. I swapped to `not config.version_lt` expression everywhere so when I was changing the logic from normally-closed to normally open. This was

a) To emphasise that "is_supported _not_ when version less than ..."
b) if check fails for some reason (like version doesn't parse), and the check return `False`, then we still run it.